media/css/mozorg/technology.scss

```diff
@@ -878,11 +807,11 @@ html[dir="rtl"] #technologies > section {
         }
 
         p {
-            @include font-size(16px);
+            @include font-size-level5;
```

Comment:
I wish this wasn't so akward with Zilla :/ 

This looks fine on desktop, but on mobile viewports this goes to `14px`, which looks about `12px` feels a little tiny. On the other hand, if we set this to `font-size-level4` and bump everything else up accordingly, it feels a little too large on desktop...

After mulling it over a bit, I think what you have done here might be the better option - but thought I'd flag it in case you had any further thoughts.